config/error.js

```diff
@@ -0,0 +1,5 @@
+const ErrorMessages = {
+  WEET_API_ERROR: 'Error al realizar la peticion de Weet'
```

Comment:
By standard the messages must be in English.

Reply:
Sure, I thing the better text is "Weet api request is failed".